helm_deploy/hmpps-prisoner-to-nomis-update/values.yaml

```diff
@@ -103,7 +103,7 @@ generic-service:
     SPRING_PROFILES_ACTIVE: "logstash"
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
     FEATURE_EVENT_DELETEALL: false
-    REPORTS_CONTACT-PERSON_PROFILE-DETAILS_RECONCILIATION_PAGE-SIZE: 100
+    REPORTS_CONTACT-PERSON_PROFILE-DETAILS_RECONCILIATION_PAGE-SIZE: 1000
     REPORTS_CONTACT-PERSON_PROFILE-DETAILS_RECONCILIATION_PARALLEL-JOBS: 6
```

Comment:
After doing some testing on T3 I think these are the optimal settings to keep all threads busy without overwhelming NOMIS